decidim-core/app/views/decidim/messaging/conversations/_new_conversation_button.html.erb

```diff
@@ -12,7 +12,7 @@
         <%= t("decidim.shared.confirm_modal.cancel") %>
       </button>
       <button type="submit" class="button button__sm md:button__lg button__secondary">
-        <%= t("next", scope: "decidim.messaging.conversations.index") %>
+        <%= t("save", scope: "decidim.messaging.conversations.index") %>
```

Comment:
In this case it makes sense to have the Next IMO. Mind that when you click the button you're actually sent to a new page with the new conversation 

![Screenshot of the new conversation modal](https://github.com/decidim/decidim/assets/717367/153343ad-d167-41b9-b60e-a6bddacee637)

```suggestion
        <%= t("next", scope: "decidim.messaging.conversations.index") %>
```

Reply:
I agree here!